routes/api/users.js

```diff
@@ -1,18 +1,18 @@
-const express = require('express');
+const express = require("express");
```

Comment:
I didn't realize it at the time but my prettier extension changed all the single quotes to double quotes.